GETTING_STARTED.md

```diff
@@ -68,70 +77,140 @@ Elasticsearch will need to be on [version 5](https://www.elastic.co/guide/en/ela
 * [dp-import-tracker](https://github.com/ONSdigital/dp-import-tracker)
+  * use `$ make debug`
 * [dp-dimension-extractor](https://github.com/ONSdigital/dp-dimension-extractor)
+  * use `$ make debug ENCRYPTION_DISABLED=true`
 * [dp-dimension-importer](https://github.com/ONSdigital/dp-dimension-importer)
+  * use `$ make debug`
 * [dp-observation-extractor](https://github.com/ONSdigital/dp-observation-extractor)
+  * use `$ make debug ENCRYPTION_DISABLED=true`
 * [dp-observation-importer](https://github.com/ONSdigital/dp-observation-importer)
+  * use `$ make debug`
 * [dp-hierarchy-builder](https://github.com/ONSdigital/dp-hierarchy-builder)
+  * use `$ make debug`
 * [dp-search-builder](https://github.com/ONSdigital/dp-search-builder)
+  * use `$ make debug`
 
 #### Filter journey:
 * [dp-search-api](https://github.com/ONSdigital/dp-search-api)
+  * use `$ make debug`
 * [dp-code-list-api](https://github.com/ONSdigital/dp-code-list-api)
+  * use `$ make debug`
 * [dp-hierarchy-api](https://github.com/ONSdigital/dp-hierarchy-api)
+  * use `$ make debug`
 * [dp-filter-api](https://github.com/ONSdigital/dp-filter-api)
+  * use `$ make debug`
 * [dp-dataset-exporter](https://github.com/ONSdigital/dp-dataset-exporter)
+  * use `$ make debug`
 * [dp-dataset-exporter-xlsx](https://github.com/ONSdigital/dp-dataset-exporter-xlsx)
+  * use `$ ./run.sh`
 * [dp-download-service](https://github.com/ONSdigital/dp-download-service)
+  * use `$ make debug`
+
+[dp-api-router](https://github.com/ONSdigital/dp-api-router) locally as well.
+  * use `$ make debug`
 
-Make sure other apps are on CMD branches, optionally try and run the [dp-api-router](https://github.com/ONSdigital/dp-api-router) locally as well.
 
 The majority of these apps will run with `make debug`, except the XLSX exporter which requires `./run.sh`
+
+### iTerm2 setup
+  - Any terminal can be used however there are advantages of using iTerm2 such as badges, profiles, arrangements, panes ability to send commands to all panes.
+  - iTerm2 can be installed here: https://www.iterm2.com/
+  - Setup the pane arrangements and profiles by following this tutorial https://blog.andrewray.me/how-to-create-custom-iterm2-window-arrangments/
+  - It can be advantageous to give each profile a badge, instructions to complete this can be found here: https://www.iterm2.com/documentation-badges.html
+
+#### Configure
+* In Zebedee file `run.sh`, remove the following line: `export SERVICE_AUTH_TOKEN="fc4089e2e12937861377629b0cd96cf79298a4c5d329a2ebb96664c88df77b67"`
+* Service authentication token creation steps can be found here: https://github.com/ONSdigital/zebedee/#service-authentication-with-zebedee
+Note that when the first login to a Florence account is detected, a mandatory password update is required. Be aware that this token expires every so often, so it might be beneficial to do the following:
+  1. Add the following to your `~/.bash_profile` replacing `<password>` with the password of the account
```

Comment:
Aren't these helper functions a duplicate of whats in the #service-authentication-with-zebedee docs? Do we need both?

How often do we think this is actually gonna be needed? If you run getAT and setAT once, then won't that be valid until you wipe your zebedee-content? If its just to make that one off process easier then fine, but if people are having to do this frequently that seems wrong - I've only ever had to set this once.

(It's possible that because I don't work with Zebedee very often I'm not familiar with what causes this to be necessary)

Reply:
Also - if this is genuinely a process that we need then super glad its well documented and totally accept that it's not for this PR to fix a bug in our process - a separate discussion may be needed.

Reply:
Updated